### PR TITLE
fix(curriculum): clarify instruction regarding color

### LINKED ISSUE #44676: RWD Beta: Step 90 - Incomplete Instructions (Learn CSS Colors by Building a Set of Colored Markers)
Hi!

In Step 90 of the "Learn CSS Colors by Building a Set of Colored Markers" course, there is a piece of information missing from the instructions, which regards adding the color red to the box-shadow property. As a newbie learner, I could deduce the correct answer and complete the exercise, but perhaps updating the instructions would avoid any future confusion by other people.

**Link:** https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-colors-by-building-a-set-of-colored-markers/step-90

## Screenshots

Step 90 - Missing Info
How it looks by following the instructions given.

## System

- Device: Laptop
- OS: Windows 10
- Browser: Google Chrome

### PR DESCRIPTION
**Pull request by DarshanVaishya**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44676

Issue #44676 states that the instruction does not clarify what color should the box-shadow have and is only shown in the hints. So I have added it in the instructions to make it more clear.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-090.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-90
 
 Now that you're familiar with the `box-shadow` property you can finalize the shadows, starting with the one for the red marker.
 
-In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, and `spreadRadius` is `0`.
+In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, `spreadRadius` is `0` and `color` is `red`.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-090.md:12]:
```suggestion
In the `.red` CSS rule, update the values for the `box-shadow` property so `offsetX` is `0`,`offsetY` is `0`, `blurRadius` is `20px`, `spreadRadius` is `0`, and `color` is `red`.
```
Nitpicky, but I'm a huge proponent of the Oxford comma. 🙂

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-090.md (hunk 1 above)
Comment (DarshanVaishya) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-090.md:12]: np, will commit that change.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Comment by DarshanVaishya** [comment]
I forgot to fetch the lint fix commit. I merged it now, so it should be good.

**Review by raisedadead — APPROVED** [review]

**Comment by gikf** [comment]
Hi @DarshanVaishya. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**